Framework/scripts/getSamplesCfg.sh

```diff
@@ -3,7 +3,8 @@
 cfgDir=${CMSSW_BASE}/src/Framework/Framework/cfg
 
 #Default options
-sampleSCfgFile="sampleSets_v6.cfg"
+#sampleSCfgFile="sampleSets_v6.cfg"
```

Comment:
I don't think we are ready to make this the default yet.

Reply:
yeah I just needed for ISR gen matching

Reply:
Then I can't merge this PR in with this change.

Reply:
Do you want me re do that change to merge? Or you will merge later ?

Reply:
It is up to you.